tools/toolchains/mbed_toolchain.py

```diff
@@ -929,15 +929,11 @@ def add_linker_defines(self):
         flags2params = {}
         if self.target.is_PSA_non_secure_target:
             flags2params = {
-                "MBED_ROM_START": "target.non-secure-rom-start",
```

Comment:
So how will these parameter be passed to the linker?

Reply:
With other code that was duplicating this define. It was duplicated this is the one that's no longer needed.